pkg/virt-handler/node-labeller/node_labeller_test.go

```diff
@@ -39,7 +39,7 @@ import (
 	v1 "kubevirt.io/api/core/v1"
 
 	"kubevirt.io/kubevirt/pkg/testutils"
-	util "kubevirt.io/kubevirt/pkg/virt-handler/node-labeller/util"
+	"kubevirt.io/kubevirt/pkg/virt-handler/node-labeller/util"
```

Comment:
Please add test case where you run one loop with old models and then second with new.

Reply:
I’m not sure I understand. Could you please elaborate on which scenario you’re trying to cover here, and which loop you are referring to?

Reply:
I would like to mimic the issue we fixing, so have a node object with the old objects and then running the loop of node-labeller expecting that we do re/write labels